### PR TITLE
Add 3 Sites

### PR DESCRIPTION
**Pull request by Helixoo**
Add huggingface, onlinegdb and blackbox ai

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please place them in alphabetical order.

**Comment by Helixoo** [comment]
I changed

File: data/Artificial_Intelligence.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 | [Bing Chat](https://bing.com) | You can ask complex questions, find comprehensive answers, get summarized information, find inspiration to build upon all in a side-by-side view, with no need to flip between tabs. |
 | [ChatGPT](https://chat.openai.com) | ChatGPT is an artificial intelligence chatbot developed by OpenAI and launched in November 2022. It is built on top of OpenAI's GPT-3 family of large language models. |
 | [Google Gemini](https://gemini.google.com) | Google Gemini is a assistant like ChatGPT, but you can upload photos and for free and it can access the internet. Successor of Google Bard. |
+| [Hugging Face](https://huggingface.co) | Hugging Face is a platform that allows you to discover, create and develop language models in artificial intelligence. |
```
Comment (wdhdev) [inline data/Artificial_Intelligence.md:9]:
```suggestion
| [Hugging Face](https://huggingface.co) | The platform where the machine learning community collaborates on models, datasets, and applications. |
```

File: data/Code_Editors.md [hunk 2]
```diff
@@ -12,7 +12,8 @@
 | [Komodo Edit](https://www.activestate.com/products/komodo-edit) | A simple yet powerful code editor, for Windows, macOS and Linux. |
 | [Nano](https://www.nano-editor.org) | A terminal-based text editor. |
 | [Notepad++](https://notepad-plus-plus.org) | A free and open-source text and source code editor for use with Microsoft Windows. |
+| [OnlineGDB](https://www.onlinegdb.com) | Online compiler and Debugger for c/c++ and other languages. |
```
Comment (wdhdev) [inline data/Code_Editors.md:15]:
```suggestion
| [OnlineGDB](https://www.onlinegdb.com) | Online compiler and debugger for C/C++ and other languages. |
```

File: data/Code_Generation.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 | Website | Description |
 |:-:|-|
+| [Blackbox](https://www.blackbox.ai) | Blackbox is an ai that is successful in generating code. |
```
Comment (wdhdev) [inline data/Code_Generation.md:5]:
```suggestion
| [Blackbox](https://www.blackbox.ai) | AI code generation, chat and search. |
```

**Review by wdhdev — APPROVED** [review]

**Pull request merged**